src/test/ui/issues/issue-81220.rs

```diff
@@ -0,0 +1,48 @@
+// tests suggestions for full path to field
+
+// run-rustfix
+
+struct Foo {
+  first : Bar,
+  second : u32,
+  third : u32,
+}
```

Comment:
You might want to run `rustfmt` on this file. The formatting style is slightly unusual (we don't put a space before `:` in any situation), which is fine, but I think `tidy` will complain about that as well. It also helps people when navigating the codebase because if it is always consistent and suddenly they find a file with an inconsistent syntax the first thought is that the "weird" style is part of the trigger for the test.